app/globals.css

```
@@ -823,63 +823,64 @@
   background: linear-gradient(90deg, #a78bfa, #38bdf8);
   -webkit-background-clip: text;
   background-clip: text;
   -webkit-text-fill-color: transparent;
 }
 
-/* zoom */
+/* zoom — 0~18% 동작, 나머지 휴지 (총 4s) */
 @keyframes title-zoom {
-  0%, 100% { transform: scale(1); filter: drop-shadow(0 0 2px currentColor); }
-  50%       { transform: scale(1.18); filter: drop-shadow(0 0 8px currentColor) drop-shadow(0 0 16px currentColor); }
+  0%, 18%, 100% { transform: scale(1); filter: drop-shadow(0 0 2px currentColor); }
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)